kuma/static/js/wiki-compat-trigger.js

```diff
@@ -32,6 +32,20 @@
                 cache: true
             }).then(function() {
                 $('.bc-table').mozCompatTable();
+
+                if (waffle.flag_is_active('bc-signals')) {
```

Comment:
nit: I'd use `window.waffle` here, since waffle isn't actually declared as a variable.

Also, and more importantly, this code will fail on our React-based pages because waffle flags are no longer emitted into the HTML and are instead fetched via an API after the page loads.